cloud/src/recycler/checker.cpp

```diff
@@ -1758,4 +1764,106 @@ int InstanceChecker::do_mow_job_key_check() {
     return 0;
 }
 
+int InstanceChecker::do_restore_job_check() {
```

Comment:
seems nothing has been checked but statistics the numbers of KVs?
we should do some actual check for consistency and integrity such as:
1. if a tablet job presents with recycling/abored, we may check it with the restore job state in FE 
2. if a job is completed we should also do some check as 1 does to make sure consistent

Reply:
With a tablet restore job in MS is hard to find the actual restore job in FE.
And dropped and completed is not stable because it may trigger recycled in any time.
And recycling state is also hard to check integrity because the kv and data is recycling.
May be we can add a recycled state? and check the data leak in recycled state.